app/src/main/AndroidManifest.xml

```diff
@@ -157,21 +157,22 @@
     <activity android:name=".app.testing.TopicRevisionTestActivity" />
     <activity
       android:name=".app.topic.questionplayer.QuestionPlayerActivity"
+      android:label="@string/question_player_activity_title"
       android:theme="@style/OppiaThemeWithoutActionBar" />
     <activity
       android:name=".app.topic.revisioncard.RevisionCardActivity"
       android:theme="@style/OppiaThemeWithoutActionBar" />
     <activity
       android:name=".app.topic.TopicActivity"
-      android:theme="@style/OppiaThemeWithoutActionBar"
-      android:label="@string/topic_page" />
+      android:label="@string/topic_page"
+      android:theme="@style/OppiaThemeWithoutActionBar" />
     <activity
       android:name=".app.walkthrough.WalkthroughActivity"
       android:theme="@style/OppiaThemeWithoutActionBar" />
     <provider
       android:name="androidx.work.impl.WorkManagerInitializer"
       android:authorities="${applicationId}.workmanager-init"
       android:exported="false"
-      tools:node="remove"/>
+      tools:node="remove" />
   </application>
-</manifest>
+</manifest>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
@rt4914 Done.